libs/vkd3d/cache.c

```diff
@@ -239,6 +239,11 @@ static void d3d12_pipeline_library_cleanup(struct d3d12_pipeline_library *pipeli
     pthread_mutex_destroy(&pipeline_library->mutex);
 }
 
+static void d3d12_pipeline_library_set_name(struct d3d12_pipeline_library *pipeline_library, const char *name)
+{
+    WARN("d3d12_pipeline_library_set_name not implemented\n");
```

Comment:
This should be FIXME.

Reply:
We have no Vulkan objects to name here, so I don't think we should spam the log with useless messages at all. Can we just ignore the name GUID for these objects?